app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -402,6 +407,25 @@ class MessagingService : FirebaseMessagingService() {
         }
     }
 
+    private fun handleChronometer(
+        builder: NotificationCompat.Builder,
+        data: Map<String, String>
+    ) {
+        val notificationWhen = data[WHEN]?.toLong() ?: 0
```

Comment:
So one of the existing features we offer is `timeout` which currently takes time in seconds and later on we do the conversion to milliseconds. Do we want to keep things consistent here?

Reply:
Updated to treat this as seconds

Reply:
So I apologize, when I commented on this I didn't realize the time in milliseconds was based on a date rather than a duration.  So in this case it is indeed different and you were correct from the beginning.  Lets wait for the official review before changing it.  Sorry for any inconvenience :)